.travis.yml

```diff
@@ -15,6 +15,18 @@ services:
 
 matrix:
   include:
+    - compiler: "gcc"
+      dist: xenial
+      env:
+        - ARCH=arm
```

Comment:
if you change this to `OS_TYPE=qemu_arm` and define a `handle_qemu_arm()` function in `./CI/travis/before_install_linux` and another one in `./CI/travis/make_linux`

moving the code from ` ./CI/travis/setup_qemu_for_arm.sh` to `./CI/travis/before_install_linux is a bit much; so `handle_qemu_arm()`  could just call or include `./CI/travis/setup_qemu_for_arm.sh`

Reply:
Thanks for review - I can try this later tonight.